guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

```diff
@@ -16,7 +16,8 @@ For more information on manifests and repositories, see {ContentManagementDocURL
 
 .Proxy and Network Prerequisites
 * The {ProjectServer} base operating system must be able to resolve the host name of the {SmartProxy} base operating system and vice versa.
-* The base operating system on which you want to install {SmartProxyServer} must not be configured to use a proxy to connect to the Red Hat CDN.
+* Ensure HTTPS client certificate authentication is possible between {SmartProxyServer} and {ProjectServer}.
+HTTP proxies often interfere with this.
```

Comment:
This sounds very vague. I'd recommend rewording to "Ensure HTTP proxies do not interfere with ..." or similar.

Reply:
I'm not sure that would cover it. Perhaps `HTTPS connection using client certificate authentication` is better than `HTTPS client certificate authentication`? That also implies network connectivity and all.

Reply:
Addressed @ekohl's suggestion above.

Reply:
I wonder if our wording should be stronger to make it clear we do not support putting a proxy between a Foreman and a Smart Proxy.

Reply:
Might I suggest:

```
HTTP proxies between {SmartProxyServer} and {ProjectServer} are not supported.
```

@ekohl I'd like your opinion on the wording.

Reply:
That sounds good to me.

Reply:
Thanks @ekohl @ehelms 

I am replacing the second sentence with the above one.